functions/exercise4/scriptSpec.js

```diff
@@ -2,6 +2,10 @@
 /*globals validateAsPhraseConsistsOfLettersOnly*/
 
 describe("Check if phrase consists of letters only.", function () {
```

Comment:
Dot at the end of description is incorrect, please replace it with : to improve reading of tests description.